src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -4,6 +4,8 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+# pylint: disable=trailing-whitespace
```

Comment:
@troydai would kill me if I let you let this through. 👎

Reply:
I disabled this because trailing whitespace was the only way I could get the help for some of the new commands to format properly.  Is there another way to force line endings in the help content? Even when I put newlines in the help strings it would still pull the content back to the old line.

Reply:
Use `|` instead of `>` in your long-summary YAML.